apps/test/unit/code-studio/pd/professional_learning_landing/EnrolledWorkshopsTest.jsx

```diff
@@ -112,4 +137,19 @@ describe('EnrolledWorkshops', () => {
       )
     );
   });
+
+  it('Clicking "Print Certificate" does not open the certificate if user did not attend workshop', function() {
+    const enrolledWorkshopsTable = shallow(
+      <EnrolledWorkshopsTable workshops={workshops} />
+    );
+
+    // Click the "Print Certificate" button
+    const printCertificateButton = enrolledWorkshopsTable
+      .find('tbody tr')
+      .at(3)
+      .find('Button')
+      .first();
+
+    expect(printCertificateButton.prop('disabled')).to.be.true;
```

Comment:
The fact that `disabled` wasn't respected on this simulated button click may be by design - in particular, we're shallow-rendering here, and using a bootstrap `Button` component, so in theory we're not even rendering the DOM element that would normally enforce `disabled` as a behavior.

I think your approach here, checking the prop itself, is fine.  Just make sure to update the comment on line 146 since we're not actually clicking the button anymore.